lib/fs.js

```diff
@@ -54,7 +54,7 @@ const {
 
 const { FSReqCallback, statValues } = binding;
 const internalFS = require('internal/fs/utils');
-const { getPathFromURL } = require('internal/url');
```

Comment:
That's a great point.

Reply:
While backporting is a concern, I would rather rip the band-aid for once and for all rather than leaving an unfamiliar function name behind for future contributors.